be/src/common/config.h

```diff
@@ -743,6 +743,9 @@ CONF_Int32(object_pool_buffer_size, "100");
 // ParquetReaderWrap prefetch buffer size
 CONF_Int32(parquet_reader_max_buffer_size, "50");
 
+// For continuous versions that rows less than small_version_max_rows will  trigger compaction quickly
+CONF_Int32(small_version_max_rows, "1000");
```

Comment:
You'd better to add a config to turn on/off this feature

Reply:
> You'd better to add a config to turn on/off this feature

set 0 means turn off this feature